src/main/resources/application.yml

```diff
@@ -102,3 +102,9 @@ allowed.origins:
 
 MICROSOFT_GRAPH_URL: https://graph.microsoft.com/v1.0
 UNLEASH_URL: https://unleash.nais.io/api/
+
+KAFKA_BOOTSTRAP_SERVERS: localhost:9092
+KAFKA_GROUP_ID: klage-oppgave-api
+EGENANSATT_KAFKA_TOPIC: aapen-skjermede-personer-v1
```

Comment:
egenansatt == skjermet person? Bare for å være sikker på min forståelse :)

Reply:
Ja. Litt snålt navnevalg på topicen, synes jeg. Men men..